packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/globalButtonsFrame.st

```diff
@@ -2,8 +2,8 @@ building
 globalButtonsFrame
 	
 	^ LayoutFrame new
-		leftFraction: 0 offset: 0;
-		topFraction: 0 offset: 0;
-		rightFraction: 1 offset: 0;
-		bottomFraction: 0 offset: self buttonHeight;
+		leftFraction: 0;
+		topFraction: 0;
+		rightFraction: 0.15;
```

Comment:
Should this value be changed to a class var or something similar? I see that we use the same value in **testingEntityFrame** as well.